code/modules/mob/living/simple_animal/bot/floorbot.dm

```diff
@@ -108,16 +115,16 @@
 			remote_disabled = !remote_disabled
 		if("autotile")
 			autotile = !autotile
-		if("replacetiles")
-			replacetiles = !replacetiles
-		if("eattiles")
-			eattiles = !eattiles
-		if("maketiles")
-			maketiles = !maketiles
+		if("replace_tiles")
+			replace_tiles = !replace_tiles
+		if("eat_tiles")
+			eat_tiles = !eat_tiles
+		if("make_tiles")
+			make_tiles = !make_tiles
 		if("nagonempty")
 			nag_on_empty = !nag_on_empty
-		if("fixfloors")
-			fixfloors = !fixfloors
+		if("fix_floor")
+			fix_floor = !fix_floor
```

Comment:
You'll need to update the js file for this, wont you?

Reply:
yes, mistakes of using replace
ty